manifests/service.pp

```diff
@@ -1,5 +1,41 @@
 # Takes care of starting heka service
 class heka::service {
+
+  case $heka::service_provider {
```

Comment:
I consider managing files and directories as part of the config class. Any objections to move this block to config.pp?